src/main/java/lesson/lesson2/Task1/HeavyPlane.java

```diff
@@ -0,0 +1,14 @@
+package lesson.lesson2.Task1;
+
+public class HeavyPlane extends Plane {
+
+
+    public HeavyPlane(String brand, int capacity, int tonnage, int range) {
+        super(brand, capacity, tonnage, range);
+    }
+
+    @Override
+    public void fly() {
+        System.out.println("Heavy plane is flying");
+    }
```

Comment:
It's not a mistake but I think you should describe more differences between plans, not only overriding fly method